### DISCUSSION TITLE
How to increase env_config.bitcoin_lock_confirmed_timeout?

### DISCUSSION THREAD

**Discussion by jadsilva** [comit-network/xmr-btc-swap#1563] [Q&A]
I think default bitcoin_lock_confirmed_timeout is 120 minutes. Its correct?
How to increase this?

**Comment by delta1** [comment]
You would have to edit this line and recompile the source: 

https://github.com/comit-network/xmr-btc-swap/blob/c3b474d7db598f3afb6a5ade8264ac0a1f5d0ece/swap/src/env.rs#L50

**Reply by jadsilva** [reply]
is it worth having this configurable?

**Reply by delta1** [reply]
Probably, but I think the real issue is that the feerate of the of the lock transaction is too low.

**Reply by delta1** [reply]
created a new issue for this https://github.com/comit-network/xmr-btc-swap/issues/1565
